test/integration/master/kube_apiserver_test.go

```diff
@@ -85,6 +85,43 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestReadyz(t *testing.T) {
+	server := kubeapiservertesting.StartTestServerOrDie(t, nil, []string{"--post-start-hook-grace-period", "10s"}, framework.SharedEtcd())
+	defer server.TearDownFn()
+	defer time.Sleep(10 * time.Second)
+	t.Run("test post start hook grace period", func(t *testing.T) {
+		if server.ServerOpts.GenericServerRunOptions.PostStartHookGracePeriod.Seconds() != 10 {
```

Comment:
the options are directly mapped to flags. There is not much value of this test I fear.

Reply:
Revised.